HW_4/Makarov_V/HW_4_2.py

```diff
@@ -0,0 +1,4 @@
+list_int = [1, 2, 3, 4, 5, 6, 7, 8, 10, 15]
+for i in list_int:
```

Comment:
new_list = [ ]

for i in list_int:
    new_list.append(float(i))
print(new_list )